src/components/structures/RoomView.tsx

```diff
@@ -114,8 +114,11 @@ import { RoomSearchView } from "./RoomSearchView";
 import eventSearch from "../../Searching";
 import VoipUserMapper from "../../VoipUserMapper";
 import { isCallEvent } from "./LegacyCallEventGrouper";
+import { WidgetType } from "../../widgets/WidgetType";
+import WidgetUtils from "../../utils/WidgetUtils";
 
 const DEBUG = false;
+const PREVENT_MULTIPLE_JITSI_WITHIN = 30_000;
```

Comment:
30 seconds seems a really long time. On the other hand, when do we _ever_ want 2 jitsi widgets?

Reply:
I don't want to change the product. Currently we can have multiple Jitsi widgets in a room. Other behaviour may be considered when switching to Element Call.

30s are a best guess. Sometimes it takes a while for an event to come through :man_shrugging:

Reply:
Sounds fine. At first I thought it was too long a time, but because I think the case of really wanting a second widget is so rare, it makes sense.